### PR TITLE
Swager changes. Putting it aside from other fixes. Might need recheck…

### PR DESCRIPTION
**Pull request by tkardu**
… type of levels, biobank and modified since

### CODE REVIEW

File: docs/cristin-proxy-swagger.yaml [hunk 1]
```diff
@@ -88,6 +88,99 @@ paths:
             type: number
             default: '5'
           example: '10'
+        - name: institution
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:91]: In code, you are using `public static final String CRISTIN_INSTITUTION_ID = "cristin_institution_id";` as input param, but when adding to query you are using `private static final String CRISTIN_QUERY_PARAMETER_INSTITUTION = "institution";`

So either you should change input param to `institution` or update swagger to use `cristin_institution_id`

File: docs/cristin-proxy-swagger.yaml [hunk 2]
```diff
@@ -520,6 +613,99 @@ paths:
...
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:616]: Same here as other comment

File: docs/cristin-proxy-swagger.yaml [hunk 3]
```diff
@@ -88,6 +88,99 @@ paths:
...
+          required: false
+          schema:
+            type: string
+        - name: sort
+          in: query
+          description: here
+          required: false
+          schema:
+            type: string
+        - name: name
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:133]: What is `name` ?

File: docs/cristin-proxy-swagger.yaml [hunk 4]
```diff
@@ -88,6 +88,99 @@ paths:
...
+          in: query
+          description: Sorts on 'start_date' and/or 'end_date'. See Search and sort. Default sort order is on 'cristin_project_id' in ascending order.
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:135]: I think this belongs under param `sort`

File: docs/cristin-proxy-swagger.yaml [hunk 5]
```diff
@@ -520,6 +613,99 @@ paths:
...
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:660]: See other comment

File: docs/cristin-proxy-swagger.yaml (hunk 1 above)
Comment (torarnet) [reply docs/cristin-proxy-swagger.yaml:91]: `institution` is the param name in Cristin

File: docs/cristin-proxy-swagger.yaml (hunk 5 above)
Comment (tkardu) [reply docs/cristin-proxy-swagger.yaml:648]: Name was a dummy-template which I was feeling out for each parameter.
Fixed it now

Comment (tkardu) [reply docs/cristin-proxy-swagger.yaml:648]: moved description for sort in sort

File: docs/cristin-proxy-swagger.yaml (hunk 1 above)
Comment (tkardu) [reply docs/cristin-proxy-swagger.yaml:91]: I will use institution for my purposes.

**Review by torarnet — APPROVED** [review]

**Pull request merged**